content/en-us/build/general/intro.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Introduction"
+weight: 1001
+draft: false
+menu:
+  build:
+    parent: "phat-general"
+---
+
+## What is Phat Contract
```

Comment:
Would be nice to have a discussion with Dan about how to introduce Phat Contract on Wiki. There's a big room to improve this page. My general feeling is that it's too abstract, and focuses too much on the technical spec. It may not be friendly to general developers.

Reply:
Should we think about the introduction including the gaps in today's Web3 development stack that force most Web3 projects to use Web2 services to "glue" together different pieces of the stack? This could be a high level list of Web2 middleware compromises seen today then we can dive into how Phat Contracts can solve these problems by being the middleware needed to make a true Web3 stack possible with scalable computation.

Reply:
> Should we think about the introduction including the gaps in today's Web3 development stack that force most Web3 projects to use Web2 services to "glue" together different pieces of the stack? This could be a high level list of Web2 middleware compromises seen today then we can dive into how Phat Contracts can solve these problems by being the middleware needed to make a true Web3 stack possible with scalable computation.

I have updated the images for current DApp stack. Can you create a new PR to add these explanations?